src/v2/Apps/Order/Components/ArtworkSummaryItem.tsx

```diff
@@ -80,15 +76,13 @@ const ArtworkSummaryItem: React.FC<ArtworkSummaryItemProps> = ({
         <Text variant="sm" color="black60">
           {shippingOrigin}
         </Text>
-        {user && userHasLabFeature(user, "Avalara Phase 2") && (
```

Comment:
Is the lab feature still used after this PR? If not, we can retire it!

Reply:
@starsirius Confirmed with Roberta this will be staying behind the flag.

Reply:
Ohhh yeah you're right. Let's keep it! Thanks for double checking!